cg/cli/post_process/post_process.py

```diff
@@ -35,4 +35,25 @@ def post_process_sequencing_run(context: CGConfig, run_name: str, dry_run: bool)
     post_processing_service.post_process(run_name=run_name, dry_run=dry_run)
 
 
-post_process_group.add_command(post_process_sequencing_run)
+@post_process_group.command(name="all")
+@DRY_RUN
+@click.pass_obj
+def post_process_all_runs(context: CGConfig, dry_run: bool) -> None:
+    """Post-process all runs from the instruments."""
+    exit_success: bool = True
+    run_names: list[str] = get_all_run_names(context=context)
+    for run_name in run_names:
+        try:
+            post_processing_service: PostProcessingService = (
+                get_post_processing_service_from_run_name(context=context, run_name=run_name)
```

Comment:
This also feels backwards to me. I would prefer if we in this approach already kept the run names grouped per post processing service as in the board sketch